modules/ducktests/src/main/java/org/apache/ignite/internal/ducktest/utils/IgniteAwareApplicationService.java

```diff
@@ -60,6 +61,11 @@ public static void main(String[] args) throws Exception {
 
         app.cfgPath = cfgPath;
 
+        String connStr = jsonNode.get("thin_client_connection").asText();
```

Comment:
use Optional, otherwise we will get an Exception in other tests
`Exception in thread "main" java.lang.NullPointerException
	at org.apache.ignite.internal.ducktest.utils.IgniteAwareApplicationService.main(IgniteAwareApplicationService.java:64)`